odoo/http.py

```diff
@@ -1718,7 +1718,12 @@ def _save_session(self):
         # - It could allow session fixation attacks.
         cookie_sid = self.httprequest.cookies.get('session_id')
         if not sess.is_explicit and (sess.is_dirty or cookie_sid != sess.sid):
-            self.future_response.set_cookie('session_id', sess.sid, max_age=SESSION_LIFETIME, httponly=True)
+            if self.db:
+                max_age = self.env['ir.config_parameter'].sudo().get_param('sessions.max_inactivity_seconds', SESSION_LIFETIME)
```

Comment:
up
```suggestion
                max_age = int(self.env['ir.config_parameter'].sudo().get_param('sessions.max_inactivity_seconds', SESSION_LIFETIME))
```

Reply:
also if `int()` is indeed necessary, I wonder if we shouldn't wrap it with a `try: except ValueError` and to log a warning in case the admin entered shit in its ICP, `_save_session` is called for every authentication, I think it would be better that we do not lock people out of their db

Reply:
I logged it for both cases (this one and the previous one)